### PR TITLE
Fixes Issue #215

### PR DESCRIPTION
**Pull request by its-ChaTTy**
## Summary
 
Closes Issue #215 . Also fixes links as referred #213 & #214 but the APIs used in these sections of lecture TWP45 are broken & are showing external error.

## Checklist
 - [x] Variables, functions and comments are translated to Spanish
 - [x] Functions follow underscore notation
 - [x] Spell check done & typos fixed
 - [x] All python code is PEP8 compliant
 - [x] Test coverage with Playwright implemented; locators are Pyhton code
 - [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

image

### CODE REVIEW

File: _sources/lectures/TWP45/TWP45_2.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ En este ejercicio vamos a acceder a Reddit para obetener datos como los de la si
     import json
     
     # La url de Reddit a la que accederemos
-    url = "https://cors.bridged.cc/http://www.reddit.com/r/Python/.json"
+    url = "https://api.allorigins.win/raw?url=http://www.reddit.com/r/Python/.json"
```
Comment (sujith-rek) [inline _sources/lectures/TWP45/TWP45_2.rst:20]: CORS error still exists

File: _sources/lectures/TWP45/TWP45_2_en.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ In this exercise we are going to access Reddit to obtain data like the one in th
     import json
     
     # The Reddit URL we'll access
-    url = "https://cors.bridged.cc/http://www.reddit.com/r/Python/.json"
+    url = "https://api.allorigins.win/raw?url=http://www.reddit.com/r/Python/.json"
```
Comment (sujith-rek) [inline _sources/lectures/TWP45/TWP45_2_en.rst:20]: CORS error still exits and lecture not working

File: _sources/lectures/TWP45/TWP45_3.rst [hunk 3]
```diff
@@ -13,41 +13,47 @@ La documentación de la API de `TasteDive <https://tastedive.com/read/api>`_.
...
 
     ~~~~
-    import requests
+    import urllib.request
+    import urllib.parse
     import json
 
     api_url = "https://tastedive.com/api/similar"
-    proxy = "https://cors.bridged.cc/"
+    proxy = "https://api.allorigins.win/raw?url="
```
Comment (sujith-rek) [inline _sources/lectures/TWP45/TWP45_3.rst:30]: CORS still not working

File: _sources/lectures/TWP45/TWP45_3.rst [hunk 4]
```diff
@@ -62,7 +68,8 @@ se transforma a un diccionario de Python. Sin embargo, no es del todo legible. E
     import json
 
     api_url = "https://tastedive.com/api/similar?"
-    proxy = "https://cors.bridged.cc/"
+    proxy = "https://api.allorigins.win/raw?url="
```
Comment (sujith-rek) [inline _sources/lectures/TWP45/TWP45_3.rst:71]: here too, CORS

**Comment by its-ChaTTy** [comment]
Hey, Thanks for the review @sujith-rek, the reddit api error has been fixed. Whereas the API for TasteDive is broken & can not be fixed , https://tastedive.com/api/similar?q=red+hot+chili+peppers%2C+pulp+fiction , you can see it here (this link was taken from TasteDive API Documentation. You can see it here https://tastedive.com/read/api)

**Comment by sujith-rek** [comment]
see if you merge main into this branch to restore the original polls

**Pull request closed without merging**